2016-01-06/swagger/Metadata.json

```diff
@@ -0,0 +1,116 @@
+{
+  "parameters": [
+    {
+      "$ref": "#/parameters/bucketName"
+    },
+    {
+      "$ref": "#/parameters/zone"
+    }
+  ],
+  "put": {
+    "tags": [
+      "Metadata"
```

Comment:
Metadata is not a resource for bucket, it's a property of object. Metadata should be used in `GetObject` or `HeadObject` as described in https://docs.qingcloud.com/qingstor/api/common/metadata.